nomos-services/log/src/lib.rs

```diff
@@ -152,7 +156,9 @@ impl ServiceCore for Logger {
                     .overwatch_handle
                     .runtime()
                     .spawn(async move { task.connect().await });
-                registry_init!(layer, config.format, config.level);
+                ONCE_INIT.call_once(move || {
+                    registry_init!(layer, config.format, config.level);
+                });
```

Comment:
What is the issue with the registry?

Reply:
Running multiple Overwatch services with log service in one test case in different threads will lead to panic as tracing only allows setting global subscriber once. There is PR #577 for it, I will update that PR to feature gate this in `test`.